lib/DDG/Spice/IndeedJobs.pm

```diff
@@ -18,15 +18,27 @@ attribution github  => ['parker', 'Parker'],
             twitter => ['ourmaninjapan', 'Daniel Davis'],
             web     => ['http://daniemon.com/', 'Daniel Davis'];
 
-triggers any => "///***never trigger***///";
+triggers any => "job", "jobs";
```

Comment:
As I tried to explain, this shouldn't part of the pull request (I just reverted deep trigger to allow testing with duckpan)